### ISSUE TITLE
IA: write a brief

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#4127]
Write [a design brief](https://docs.google.com/document/d/1IZ8dVsLXrBrge5f7vJ69Gj4mMsIGQ9XlT0VgwuFwfIQ/edit#heading=h.e1hdva6khlbd) for the IA work.

**Comment by taisdesouzalessa** [comment]
Brief is written - please feel free to edit/comment to make it more complete @kristinashu @beccaklam 

Link to the [Design Brief](https://docs.google.com/document/d/1IZ8dVsLXrBrge5f7vJ69Gj4mMsIGQ9XlT0VgwuFwfIQ/edit?usp=sharing)

**Comment by kristinashu** [comment]
Thank you for doing this Tais! I've just assigned it to Rebecca for review and then we're good to close!

**Comment by beccaklam** [comment]
I've reviewed it and it looks good to me! Thanks @taisdesouzalessa I just added one note about verifying assumptions around user needs.
I think we can even be more specific with our goals after we interview stakeholders and users (?).
From the IA book I was reading I found this bit helpful:

> Whenever I need to determine criteria for organizing digital content, I look to four key factors to shape my categorization:
> 1. The needs of users
> 2. The goals of the business
> 3. The current state of the content
> 4. The strategic future of the content

In some cases we may only be able to guess a user's needs but where we can interview a real user, I think it would be insightful to hear from them.
